Package.swift

```diff
@@ -1,8 +1,8 @@
-// swift-tools-version:4.0
+// swift-tools-version:5.0
 // The swift-tools-version declares the minimum version of Swift required to build this package.
 
 /**
- * Copyright IBM Corporation 2016, 2017
+ * Copyright IBM Corporation 2019
```

Comment:
I know this is nitpicking but shouldn't this be 2016-2019?

Reply:
Thanks, fixed :)